### ISSUE TITLE
Examples for CICategoryColorEffect

### ISSUE THREAD

**Issue by noahsark769** [noahsark769/cifilter.io#51]

**Comment by duemunk** [comment]
I would love to start expanding example photos, but I'm not sure where to start. Do you have a script or tool setup to generate all the examples or have they been created manually?

**Comment by noahsark769** [comment]
@duemunk yes, there's a process! Unfortunately it's fairly manual and not documented at the moment. The process looks like this:

1. Build a debug build of the app (I think you might be the first one to do this other than me, so feel free to reach out with any questions)
2. Go into the app in debug mode, put in some parameters for the filter you'd like to test and generate an output image, then tap the *other* share button, which calls [this method](https://github.com/noahsark769/cifilter.io/blob/master/iOS/CIFilter.io/Filter%20Workshop/FilterApplicationExporter.swift#L167) on `FilterApplicationExporter`
3. This will generate a filesystem directory named with a unique UUID - download the app's container via Xcode and drop that directory into [web/src/data/examples](https://github.com/noahsark769/cifilter.io/tree/master/web/src/data/examples) and everything should work!
